### DISCUSSION TITLE
Recommended smartphones

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#599] [Show and tell]
Inspired by the Elebest question:
Are there any recommended smartphones/outdoor smartphones for motorcycles?
The things should be waterproof, have a bright display and be temperature-resistant (heating due to sunlight).
Who has good or bad experiences?

**Comment by devemux86** [comment]
Advice: go for as new Android as possible.

---

Old Androids up to 7 (Nougat) have trouble communicating with https websites.
Internet is updating the https protocol but old Androids do not receive updates.

Cruiser however can get around this. :slightly_smiling_face:

**Comment by Leeb64** [comment]
I'll start right away.
My navigation smartphone is a Sony Experia xz2.Android 10.
It is waterproof and has a bright display. Unfortunately, it becomes weaker in the heat (long periods of direct sunlight).
The new price 5 years ago was €800.
You can get it used for €80.

**Comment by umpfmpf** [comment]
I've been driving with a Doogee S88plus for 3 years now. 
New for around €150 and can easily last a day without a charger. 
However, my first one developed a slight stomach after 2.5 years and was therefore thrown away.
It has Android 10, although the S98 is already available with Android 12.
With a matt protective film you can also do it when the sun is shining.

**Comment by MotoUKRider** [comment]
I was going to start a thread on suggested/rugged devices - I'm fed up of letter-box aspect-ratio devices and would prefer a more square (but not quite 4:3) shape.  I presently use a Samsung S20 with an S9 for backup/testing.  With *Motorcycle Youtube* going ga-gaa for Auto/CarPlay remote screens right now, that seems to be where the game is: A nice screen that doesn't contain all your essential data that can be left on the bike when you stop for a coffee.  Of course, the apps have to keep up!

In terms of "rugged", at the cheap end of the scale, I've found the OUKITEL WP32 (18:9 ratio) for around £130.
There must be some more Android devices out there that sit between "Phone" and "Tablet" without breaking the bank?!

**Comment by r11d11** [comment]
I use a probably an unusual smartphone on the motorbike
I needed a thin smartphone that fit into my inductive charging cradle on the motorbike
So I use an Oppo Find X3 Pro for 2 years now on my motobike and bicicle with cruiser. I am very happy with this smartphone.
You can get it used for €250

I charge inductively. Works fine with cruiser.
Display is very bright, no problems with the sunlight at all.
Oled display not yet burnt in.
Bluetooth connection to headset very stable and loud
Waterproof according to IP68 
I haven't had any problems with overheating so far
Snapdragon 888 is quit fast
I think the camera has no image stabilisation so I have no problems because of the vibrations on the bike

Disadvantages.
It is with 6,7 Zoll relatively large
Not so robust as an outdoor device
Unfortunately, there are no further updates after Android 14. 
But that should be enough for a few more years...

And with Cruiser, I have retired my old Tomtom :-)

**Comment by GuG1050** [comment]
After retiring my Garmin already a few years ago (first replaced by Kurviger, now Cruiser) and after two disappointing experiences with a Samsung  Xcover 5  (dark and small screen) and a Gigaset GX290 (died on route) I now use a Google Pixel 6 since a year. I bought the Pixel used for 240 € to be used exclusively for motorcycle (and MTB) navigation. I am happy with it so far:
- bright screen 6.4 in
- good readability also in sunlight 
- powerful processor
- sufficient storage
- waterproof
- inductive charging (I use SP connect)
- runs Android 14 currently
- rugged enough with the SP case and not bulky

Only downside is that inductive charging seems to stop on sunny days at temperatures over 25 °C. Discovered recently in the hot weather. Seems to be a overheating protection. No big problem so far as the battery always lasted long enough for a day.

**Comment by MotoUKRider** [comment]
Re-igniting this topic: Are there any good devices worth looking at that are:

a) 4:3 or similar aspect ratio (not smartphone letter-box)
b) 6-7-inch not-quite-tablet sized
c) Reasonably-priced
d) Good screen, eg: sunlight
e) 16-32GB (and/or SD card)

Obviously it needs to run whatever Android version was released 10-seconds ago so it conforms to whatever neurotic superstitions GooglePlay has about security patches and have some sort of AMPS/rugged mount capacity. I don't mind hunting for mounts/brackets but it's mainly the actual size of the unit that is the primary concern.

Clearly, the sensible solution is an Android-Auto unit (every Motorcycle YouTuber seems to have jumped on that bandwagon) but we don't have that feature in Cruiser.

**Reply by WolfgangZ42** [reply]
> solution is an Android-Auto unit 

NO - because it always would be a subset of Cruisers offers.

**Reply by devemux86** [reply]
Android Auto has extremely strict requirements that leave very little room for app customization.
Minimum options and even the interface must follow certain rules.

Phones and tablets will always offer much more flexibility and many more options.

**Reply by MotoUKRider** [reply]
When I say "sensible", I'm talking about the most basic of simple "gadget" rather than a smartphone that *may* have stuff on it that you wouldn't want somebody else to have.  Obviously you could strip it of Apps and just use it for navigating but it'd still have your Google account on it.  It's a shame that Auto is such a mood-hoover.

There's the "faff" of photo stops and such: Disconnecting the phone easily/safely each time. There must be some good products out there (like the Touratech GPS Lock) that can secure a device while you're paying for fuel but also allow it to be removed easily. A locking mount with a wireless charging panel?

This is what I'm currently using: An Alicheap phone holder. I've removed the 2 legs at the top and the right hand bit is on a spring so you just pull it to widen the gap and slot the phone down where it rests on the bottom legs and is held by the spring.
![vstrom-phoneholder-2](https://github.com/user-attachments/assets/2ca2ff6a-2906-41ff-ac8d-8d7673c0ab32)
